op-node/service.go

```diff
@@ -89,9 +89,8 @@ func NewConfig(ctx *cli.Context, log log.Logger) (*node.Config, error) {
 		Beacon:        NewBeaconEndpointConfig(ctx),
 		InteropConfig: NewSupervisorEndpointConfig(ctx),
 		RPC: node.RPCConfig{
-			ListenAddr:  ctx.String(flags.RPCListenAddr.Name),
-			ListenPort:  ctx.Int(flags.RPCListenPort.Name),
-			EnableAdmin: ctx.Bool(flags.RPCEnableAdmin.Name),
```

Comment:
was this removed on purpose?

Reply:
No it was not! Good catch I will reinstate it.

Reply:
[d141419](https://github.com/ethereum-optimism/optimism/pull/15150/commits/d1414198a334d4a8237f98c45cf0fba7de2deabb)